api/openapi/openapi_generated.go

```diff
@@ -514,6 +514,12 @@ func schema_pkg_apis_ignite_v1alpha1_VMSpec(ref common.ReferenceCallback) common
 							Ref: ref("github.com/weaveworks/ignite/pkg/apis/meta/v1alpha1.Size"),
 						},
 					},
+					"networkMode": {
```

Comment:
rebase this to latest master and see if these changes still apply or go away...

Reply:
Rebased, those changes seem to have disappeared.